### ISSUE TITLE
manage command does not handle arguments with spaces properly

### ISSUE THREAD

**Issue by luser** [Miserlou/Zappa#1936]
## Context

Attempting to invoke a Django management command where one of the arguments contains spaces, like `zappa manage prod mycommand '\"a b c\"'` results in the argument with spaces being split.

## Expected Behavior

Arguments that are quoted should be preserved, in this case `mycommand` should be called with the argument `a b c`.

## Actual Behavior

In this case `mycommand` is called with the arguments `"a`, `b`, `c"`.

## Possible Fix

[The handler code](https://github.com/Miserlou/Zappa/blob/60fbb55fffa762a85e79e756f2a1373832d78320/zappa/handler.py#L413) is currently calling `*event['manage'].split(' ')`, but it should probably be using `shlex.split(*event['manage'])` which will do the right thing here:

```
>>> import shlex
>>> s = 'a b "c d"'
>>> s.split(' ')
['a', 'b', '"c', 'd"']
>>> shlex.split(s)
['a', 'b', 'c d']
```

## Steps to Reproduce

Run a django management command, pass an argument containing spaces, such as `zappa manage prod mycommand '\"a b c\"'`

## Your Environment
* Zappa version used: `0.48.2`

**Comment by rollcat** [comment]
A better solution (the one I suggested in #1937), would be to send the unchanged, verbatim array of strings, and decode / execute that on the remote end. It's a bit more involved than just doing a `shlex.split`, there would be changes in `cli.py` around 581-591 (take the array) and in `def invoke`, the format of the message sent to the handler would have to change to use an array of strings as opposed to a string, then finally `handler.py` at 413 would call something like `management.call_command(*event['manage_args'])`.

**Comment by luser** [comment]
I agree that that would be a better fix. I went for a simple targeted fix but keeping the arguments as a list instead of joining them and then splitting them would be ideal.
